jcapi-systemusers.go

```diff
@@ -71,6 +78,10 @@ func (jcuser JCUser) ToString() string {
 	returnVal += fmt.Sprintf("JCUSER: PasswordExpired=[%t] - Active=[%t] - PendingProvisioning=[%t]\n", jcuser.PasswordExpired, jcuser.Activated,
 		jcuser.PendingProvisioning)
 
+	if len(jcuser.Attributes) > 0 {
+		returnVal += fmt.Sprintf("JCUSER: Attributes %s", jcuser.Attributes)
```

Comment:
Implemented ToString() on JCUserAttribute. Same convention as tags in JCUser.ToString()